### PR TITLE
Update readme with new duckdns info

### PR DESCRIPTION
**Pull request by nemchik**
Following up from https://github.com/linuxserver/docker-swag/pull/277
Not adding an additional changelog entry since the previous PR includes a changelog entry mentioning the changes, but this PR adjusts the readme to reflect the changes.

Removing information about `duckdns` validation.

It was a custom implementation. It worked fine, but I'm working on making all validation consistent in more ways in the future, so using the `duckdns` plugin fits that goal.

The old duckdns variables are only being removed from the documentation, and `duckdns` is being listed as an option for `DNSPLUGIN`. The old variables will continue to work, the logic is not being removed from the scripts, just translated behind the scenes to work with the plugin.

### CODE REVIEW

File: root/etc/cont-init.d/50-certbot [hunk 1]
```diff
@@ -221,7 +221,7 @@ if [ ! "$URL" = "$ORIGURL" ] || [ ! "$SUBDOMAINS" = "$ORIGSUBDOMAINS" ] || [ ! "
     if [[ -f /config/etc/letsencrypt/live/"$ORIGDOMAIN"/fullchain.pem ]]; then
         certbot revoke --non-interactive --cert-path /config/etc/letsencrypt/live/"$ORIGDOMAIN"/fullchain.pem --server $REV_ACMESERVER
     fi
-    rm -rf /config/etc/letsencrypt/{archive,live,renewal}
+    rm -rf /config/etc/letsencrypt/{accounts,archive,live,renewal}
```
Comment (nemchik) [inline root/etc/cont-init.d/50-certbot:224]: This (and line 244) is the only substantial change not related to the readme. I believe this change solves the issue described by a user on discord https://discord.com/channels/354974912613449730/805754854982025246/1029144612687839262

File: README.md [hunk 2]
```diff
@@ -63,13 +63,15 @@ The architectures supported by this image are:
 ### Validation and initial setup
 
 * Before running this container, make sure that the url and subdomains are properly forwarded to this container's host, and that port 443 (and/or 80) is not being used by another service on the host (NAS gui, another webserver, etc.).
+* If you need a dynamic dns provider, you can use the free provider duckdns.org where the `URL` will be `yoursubdomain.duckdns.org` and the `SUBDOMAINS` can be `www,ftp,cloud` with http validation, or `wildcard` with dns validation. You can use our [duckdns image](https://hub.docker.com/r/linuxserver/duckdns/) to update your IP on duckdns.org.
```
Comment (nemchik) [inline README.md:66]: This was just moved up (from a few lines down)

**Review by aptalca — APPROVED** [review]

**Comment by xconverge** [comment]
It is kind of confusing for me to understand what I am supposed to change with my setup to have things continue to function properly

DUCKDNSTOKEN is still used but now I am not sure how since it is not mentioned in the docs, what is the ideal config/variable setup for duckdns?

I appreciate that functionality seems to be backwards compatible/maybe still working but I would like to update to whatever the ideal setup/config is and find it hard to get a clear answer in the docs so far with the changes

**Comment by xconverge** [comment]
@nemchik thanks for the reply!

I think for this particular container, I find the changelog hard to follow because it does not change every release, it keeps the latest changelog update (sometimes for a handful of container releases). I think the changelog/release notes should be cleared/reset for each release, even if they are only "Update linuxserver base image"

I am just a bit sensitive to the recent changes to the nginx base image that required a decent amount of maintainance, and having to dig a bit to understand what I need to change. I proactively was digging a bit for this one to understand what these changes actually were instead of them being handed to me in the changelog

**Comment by nemchik** [comment]
Normally if you look at the https://github.com/linuxserver/docker-swag/releases page, any time the `-ls#` suffix on the release changes you will see a new set of "LinuxServer Changes" listed on the release. This is not always the case, as we will sometimes push minor changes or corrections in quick succession. If the number in front of the `-ls#` suffix changes it indicates the primary package for the image (in SWAG's case, `certbot`) has updated, and there are likely no changes from our team.

In the case of `[-ls154](https://github.com/linuxserver/docker-swag/releases/tag/1.31.0-ls154)` and [-ls155](https://github.com/linuxserver/docker-swag/releases/tag/1.31.0-ls155), the changes in 155 (which were primarily changes to the documentation in the readme) should have been included with 154. In both 154 and 155 there is not a clear indication that users do not need to take any action (there are no breaking changes or deprecations). I understand that this may make it more difficult to hunt down what has changed (the commit history in GitHub shows everything, but can be a bit technical). We will continue to do our best to indicate in the changelogs when there's something requiring user attention in any given update. We also use https://info.linuxserver.io/ to announce some changes (ahead of time if possible) and go into more detail (such as https://info.linuxserver.io/issues/2022-08-20-nginx-base/ ) as needed.
